src/js/components/InfiniteScroll/__tests__/InfiniteScroll-test.js

```diff
@@ -75,4 +80,62 @@ describe('InfiniteScroll', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test(`Should render items equal to the length of 
+  step + 1 when step < items.length`, () => {
+    const step = 50;
```

Comment:
Just calling this out to set stage for future context:

As our test scenarios get more complex, we will likely see a couple of other marker refs being included in the array. We will also likely want to find a way to re-use and reference these three tests by other tests.

As we write those scenarios, these tests may get refactored to accommodate. That said, I think the current state helps us move forward and we can refactor as needed.